data/json/vitamin.json

```diff
@@ -3,39 +3,64 @@
     "id": "calcium",
     "type": "vitamin",
     "name": "Calcium",
-    "min": -600,
-    "deficiency": [ [ "brittlebone", -300 ] ]
+    "deficiency": "hypocalcemia",
+    "min": -720,
+    "disease": [
+      [ -288, -336 ],
+      [ -337, -384 ],
+      [ -385, -720 ]
```

Comment:
`min` and `max` are the lowest/highest levels a vitamin can be set to.

> You could also have a more meaningful name for a warning threshold than simply min.

This isn't how the code functions. Each `disease` entry sets an increasing intensity for the `deficiency` status.